.vscode/settings.json

```
@@ -0,0 +1,16 @@
+{
+  "files.associations": {
+    "Kraftfile": "yaml",
+  },
+  "editor.rulers": [80],
```

```suggestion
  "editor.rulers": [80, 100, 120],
```

I usually use these, we could at least add `100` also

---

Most of the files are 80, which I think is preferable, I fear that adding extra may incentive setting at longer markers.

---

yeah, that's why there are two: 80 - soft limit, 100 - hard limit :D